tests/functional/nfs_feature/test_nfs_feature_enable_for_ODF_clusters.py

```diff
@@ -1132,174 +1132,137 @@ def test_respin_app_pod_exported_nfs_volume_incluster(
-        # Create nfs pvcs with storageclass ocs-storagecluster-ceph-nfs
-        nfs_pvc_obj = helpers.create_pvc(
-            sc_name=self.nfs_sc,
-            namespace=self.namespace,
-            pvc_name="nfs-pvc",
-            size="5Gi",
-            do_reload=True,
-            access_mode=constants.ACCESS_MODE_RWO,
-            volume_mode="Filesystem",
-        )
+        try:
+            # Create nfs pvcs with storageclass ocs-storagecluster-ceph-nfs
+            nfs_pvc_obj = helpers.create_pvc(
+                sc_name=self.nfs_sc,
+                namespace=self.namespace,
+                pvc_name="nfs-pvc",
+                size="5Gi",
+                do_reload=True,
+                access_mode=constants.ACCESS_MODE_RWO,
+                volume_mode="Filesystem",
+            )
 
-        # Create deployment config for app pod
-        log.info("----create deployment config----")
-        deployment_config = """
-            apiVersion: apps.openshift.io/v1
-            kind: DeploymentConfig
-            metadata:
-              name: nfs-test-pod
-              namespace: openshift-storage
-              labels:
-                app: nfs-test-pod
-            spec:
-              template:
-                metadata:
-                  labels:
-                    name: nfs-test-pod
-                spec:
-                  restartPolicy: Always
-                  volumes:
-                  - name: vol
-                    persistentVolumeClaim:
-                      claimName: nfs-pvc
-                  containers:
-                  - name: fedora
-                    image: fedora
-                    command: ['/bin/bash', '-ce', 'tail -f /dev/null']
-                    imagePullPolicy: IfNotPresent
-                    securityContext:
-                      capabilities: {}
-                      privileged: true
-                    volumeMounts:
-                    - mountPath: /mnt
-                      name: vol
-                    livenessProbe:
-                      exec:
-                        command:
-                        - 'sh'
-                        - '-ec'
-                        - 'df /mnt'
-                      initialDelaySeconds: 3
-                      periodSeconds: 3
-              replicas: 1
-              triggers:
-                - type: ConfigChange
-              paused: false
-            """
-        deployment_config_data = yaml.safe_load(deployment_config)
-        helpers.create_resource(**deployment_config_data)
-        time.sleep(60)
-
-        assert self.pod_obj.wait_for_resource(
-            resource_count=1,
-            condition=constants.STATUS_RUNNING,
-            selector="name=nfs-test-pod",
-            dont_allow_other_resources=True,
-            timeout=60,
-        )
-        pod_objs = pod.get_all_pods(
-            namespace=self.namespace, selector=["nfs-test-pod"], selector_label="name"
-        )
+            # Create deployment config for app pod
```

Comment:
in OCP 4.14 DeploymentConfigs were deprecated - https://docs.openshift.com/container-platform/4.15/applications/deployments/what-deployments-are.html
Please use Deployments instead

Reply:
Removed references of deployment config from variable names and comments